py/aou_workbench_client/config.py

```diff
@@ -0,0 +1,32 @@
+"""Reads configuration settings from disk.
+"""
+
+import json
+import os
+
+# Constants for keys in the configuration
+ALL_OF_US_CONFIG_FILE = os.getenv('ALL_OF_US_CONFIG_PATH', '~/config/all_of_us_config.json')
```

Comment:
FYI: My localization doc intends to change this JSON file location to be collocated with the notebook itself. That's not in effect yet, but just for thought. Let me know if you think this is going to cause any issues.

Reply:
That's probably fine -- we'd need a new pyclient version at that point. 

That said it seemed like there might be good reasons to keep notebook files and config files separate... is there a particular reason to have them in the same dir?